website/docs/docs/build/dimensions.md

```diff
@@ -56,14 +56,18 @@ semantic_models:
   dimensions:
     - name: order_date
       type: time
-      label: "Date of transaction" # Recommend adding a label to define the value displayed in downstream tools
-      expr: date_trunc('day', ts)
+      type_params:
+        time_granularity: day
+      label: "Date of transaction" # Recommend adding a label to provide more context to users consuming the data
+      expr: ts
     - name: is_bulk_transaction
       type: categorical
       expr: case when quantity > 10 then true else false end
 ```
 
-MetricFlow requires that all dimensions have a primary entity. This is to guarantee unique dimension names. If your data source doesn't have a primary entity, you need to assign the entity a name using the `primary_entity: entity_name` key. It doesn't necessarily have to map to a column in that table and assigning the name doesn't affect query generation.
+Dimensions are bound to the primary entity of the semantic model in which they are defined. For example, if a dimension called `is_bulk_transaction` is defined in a model with `transaction` as a primary entity, then `is_bulk_transaction` is scoped to the `transaction` entity. To reference this dimension you would use the fully qualified dimension name `transaction__is_bulk_transaction`. 
```

Comment:
Might be nice to use an example dimension name that makes it somewhat clear why we bind it to the entity name. E.g., something like `transaction__country` or just changing the name to something like `transaction__is_bulk` would make this feel less redundant.

Reply:
Fixed.